README.md

```diff
@@ -1,28 +1,28 @@
 # stoa
 API server for Agora
 
+# Build instructions
+Install the Node.js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
 
-
-## Build instructions
-
-# Install the node.Js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
-
-
-# Building on Ubuntu
-# Ubuntu is already nodejs installed.
+## Building on Ubuntu
+Ubuntu is already Node.js installed.
+```sh
 $ sudo apt-get install nodejs
 $ sudo apt-get install npm
 $ npm ci
+```
 
-# Building on MacOS
+## Building on MacOS
+```sh
 $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 $ brew install node
 $ npm ci
+````
 
 ## How to start
-- `npm start`: Run the stoa in the Node.Js. It is being watched by nodemon.
+`npm start`: Run the stoa in the Node.js. It is being watched by nodemon.
```

Comment:
Likewise here, "Stoa" (Let's capitalize it like Agora) is the name of the node. Hence you say either "Run Stoa" or "Run the node".

Reply:
We will now use `Stoa`